### DISCUSSION TITLE
ios端给电脑端开热点 但是手机端检测不到电脑端的存在

### DISCUSSION THREAD

**Discussion by lpxks** [localsend/localsend#2074] [L-简体中文]
纯小白，昨天开始使用。而且我还试了安卓，这个倒是手机电脑都可以相互识别到，但是手机给电脑发文件，一直未响应直接超时了。请问各位大佬是我哪里设置不对吗？

**Comment by lpxks** [comment]
后面通过链接分享可以做到，但是还要到浏览器一个个点下载太麻烦了 有更好的解决办法吗?

**Comment by Lw-COD** [comment]
我也想问
